docs/run/max-execution-time.mdx

```diff
@@ -59,7 +59,7 @@ with Session(max_time="1h"):
     ...
 ```    
 
-Additionally, there is a 5-minute *interactive* timeout value. If there are no session jobs queued within that window, the session is temporarily deactivated and normal job selection resumes. During job selection, if the job scheduler gets a new job from the session and its maximum timeout value has not been reached, the session is reactivated until its maximum timeout value is reached.
+Additionally, there is a 1-minute *interactive* timeout value. If there are no session jobs queued within that window, the session is temporarily deactivated and normal job selection resumes. During job selection, if the job scheduler gets a new job from the session and its maximum timeout value has not been reached, the session is reactivated until its maximum timeout value is reached.
```

Comment:
I would not hard-code values in the docs. With https://github.com/Qiskit/qiskit-ibm-runtime/pull/1119 users will be able to get the interactive ttl for their session so the docs should just give a code snippet showing how to get the value that way. Applies below as well.

Reply:
I can do that, although it seems like a hassle for them.  What if I just gave them the range that this could be, and also the code snippit.  Then they could look it up to check if they really care.  

> I would not hard-code values in the docs. With [Qiskit/qiskit-ibm-runtime#1119](https://github.com/Qiskit/qiskit-ibm-runtime/pull/1119) users will be able to get the interactive ttl for their session so the docs should just give a code snippet showing how to get the value that way. Applies below as well.